Framework/Core/include/Framework/ArrowTypes.h

```diff
@@ -63,18 +63,23 @@ struct arrow_array_for<double> {
 };
 template <int N>
 struct arrow_array_for<float[N]> {
-  using type = arrow::FixedSizeBinaryArray;
+  using type = arrow::FixedSizeListArray;
+  using elemtype = float;
 };
 template <int N>
 struct arrow_array_for<int[N]> {
-  using type = arrow::FixedSizeBinaryArray;
+  using type = arrow::FixedSizeListArray;
+  using elemtype = int;
 };
 template <int N>
 struct arrow_array_for<double[N]> {
-  using type = arrow::FixedSizeBinaryArray;
+  using type = arrow::FixedSizeListArray;
+  using elemtype = double;
```

Comment:
in general `std::` use `value_type` for items in e.g. a vector. Maybe it makes sense to keep the convention?

Reply:
done